crypto/key.go

```diff
@@ -433,6 +433,13 @@ func (key *Key) ToPublic() (publicKey *Key, err error) {
 	return
 }
 
+func (key *Key) isVersionSix() bool {
```

Comment:
Normally I'm in favor of writing things out fully, but for this we normally refer to them as "V6" so I would call this `isV6`, also for consistency with go-crypto's `UpgradeToV6` etc.